test/e2e/functional_test.go

```diff
@@ -835,6 +835,25 @@ spec:
 		})
 }
 
+func (s *FunctionalSuite) TestDataTransformation() {
+	s.Need(fixtures.BaseLayerArtifacts)
+	s.Given().
+		Workflow("@testdata/data-transformation.yaml").
+		When().
+		SubmitWorkflow().
+		WaitForWorkflow(2 * time.Minute).
```

Comment:
this is too long for an e2e test - can you please make it shorter?

Reply:
Done